frontend/src/components/AdminDash/index.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import styles from '../../styles/AdminDash.module.css';
+import ApplicantList from './ApplicantList';
+import ApplicantInfo from './AppInfo';
+import { Button, Input } from 'reactstrap';
+import Filters from './Filters';
+
+const dummyUsers = [
+  {
+    name: 'John Adams',
+    email: 'jadams76@foundingfathers.gov',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Henry Clay',
+    email: 'greatcompromiser@unelectable.edu',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Benjamin Franklin',
+    email: 'tobefrankwithyou@unsungheros.net',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Alexander Hamilton',
+    email: 'bigbanks@dieyoung.com',
+    role: 'Volunteer'
+  },
+  {
+    name: 'James K. Polk',
+    email: 'napoleonofthestump@forgotten.gov',
+    role: 'Pending'
+  },
+  {
+    name: 'John Adams',
+    email: 'jadams76@foundingfathers.gov',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Henry Clay',
+    email: 'greatcompromiser@unelectable.edu',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Benjamin Franklin',
+    email: 'tobefrankwithyou@unsungheros.net',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Alexander Hamilton',
+    email: 'bigbanks@dieyoung.com',
+    role: 'Volunteer'
+  },
+  {
+    name: 'James K. Polk',
+    email: 'napoleonofthestump@forgotten.gov',
+    role: 'Pending'
+  },
+  {
+    name: 'John Adams',
+    email: 'jadams76@foundingfathers.gov',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Henry Clay',
+    email: 'greatcompromiser@unelectable.edu',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Benjamin Franklin',
+    email: 'tobefrankwithyou@unsungheros.net',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Alexander Hamilton',
+    email: 'bigbanks@dieyoung.com',
+    role: 'Volunteer'
+  },
+  {
+    name: 'James K. Polk',
+    email: 'napoleonofthestump@forgotten.gov',
+    role: 'Pending'
+  }
+];
+
+export default class AdminDash extends Component {
+  constructor() {
+    super();
+    this.state = {
+      selectedApplicantIndex: 0,
+      showFilterModal: false
+    };
+    this.onSelectApplicant = this.onSelectApplicant.bind(this);
+    this.onShowFilterModal = this.onShowFilterModal.bind(this);
+  }
+  onSelectApplicant(index) {
+    this.setState({
+      selectedApplicantIndex: index
+    });
+  }
+  onShowFilterModal() {
+    this.setState({
+      showFilterModal: !this.state.showFilterModal
+    });
+  }
+  render() {
+    const { selectedApplicantIndex, showFilterModal } = this.state;
+    return (
+      <div className={styles.container}>
+        <h1 style={{ fontSize: '1.4em', padding: '1rem 2rem' }}>Admin Dashboard</h1>
```

Comment:
inline CSS should probably be moved to the css module?

Reply:
AH forgot to remove this! I had it to experiment with h1 tags in general, and wanted to move that to a global definition for h1.